packages/react-strict-dom/tests/css-test.native.js

```diff
@@ -1248,8 +1250,85 @@ describe('properties: custom property', () => {
           hover: true
         },
         underTest
-      )
-    ).toMatchSnapshot();
+      ).style.color
+    ).toBe('#333');
+  });
+
+  test('rgb(a) function with args applied through a single var', () => {
+    const customProperties = { example: '24, 48, 96' };
+    expect(
+      resolveCustomPropertyValue(customProperties, [
+        'color',
+        'rgb(var(--example))'
+      ])
+    ).toEqual('rgb(24, 48, 96)');
+    expect(
+      resolveCustomPropertyValue(customProperties, [
+        'color',
+        'rgba(var(--example), 0.5)'
+      ])
+    ).toEqual('rgba(24, 48, 96, 0.5)');
+  });
+
+  test('rgba function with args applied through multiple (& nested) vars', () => {
+    const customProperties = {
+      red: 255,
+      green: 96,
+      blue: 16,
+      rgb: 'var(--red), var(--green), var(--blue)',
+      alpha: 0.42
+    };
+    expect(
+      resolveCustomPropertyValue(customProperties, [
+        'color',
+        'rgba(var(--rgb), var(--alpha))'
+      ])
+    ).toEqual('rgba(255, 96, 16, 0.42)');
+  });
+
+  test('text shadow with nested/multiple vars', () => {
+    const customProperties = {
+      height: '2px',
+      width: '1px',
+      radius: '3px',
+      red: 'red'
+    };
+    const styles = css.create({
+      test: {
+        textShadow: 'var(--width) var(--height) var(--radius) var(--red)'
+      }
+    });
+    expect(
+      css.props.call({ ...mockOptions, customProperties }, styles.test).style
+    ).toStrictEqual({
+      textShadowColor: 'red',
+      textShadowOffset: {
+        height: 2,
+        width: 1
+      },
+      textShadowRadius: 3
+    });
+  });
+
+  test('css variable declaration inside a media query', () => {
+    const customProperties = {
+      example: '42px'
+    };
+    const styles = css.create({
+      test: {
+        '@media (min-width: 400px)': {
```

Comment:
I know there's a new syntax but is this previous syntax entirely unsupported? I added this test because the logic for handling this syntax was never removed from the polyfill before and I wanted to maintain functional parity in every other part other than css variable resolution.

Reply:
It does look like we're missing unit tests for the new MQ syntax in general. The implementation probably hasn't been implemented yet. Supporting the legacy syntax on native isn't essential